controllers/operands/kubevirtConsolePlugin.go

```diff
@@ -183,7 +183,7 @@ http {
 		}
 	}
 `
-	nginxConfig = strings.ReplaceAll(nginxConfig, "$SERVER_PORT", string(hcoutil.UiPluginServerPort))
+	nginxConfig = strings.ReplaceAll(nginxConfig, "$SERVER_PORT", fmt.Sprint(hcoutil.UiPluginServerPort))
```

Comment:
Also, please consider to store the ngnixConfig outside of the function, because its value does not depand in any runtime information, and it can be pre-calculated.

Reply:
sure, makes sense. done.